pkg/application/application.go

```diff
@@ -52,3 +55,28 @@ func Create(name string) error {
 
 	return nil
 }
+
+// GetCurrentApplication gets currently active application
+// if no application is active, defaultApplication is created
+func GetCurrentApplication() (string, error) {
```

Comment:
I think should be a part of the config package in config.go, like we have other methods there like, `ApplicationExists()`, `AddApplication`, we can also have `CurrentApplication()`

Reply:
I think that if it is operation on application that it should be in the application package. 
Doing it like this, if we move to another way to store application info we won't have to touch anything else than this one function